src/commands/config-get.json

```
@@ -1,7 +1,7 @@
 {
     "GET": {
         "summary": "Get the values of configuration parameters",
-        "complexity": "PATCH__TBD__45__",
+        "complexity": "O(N) when N is the number of configuration parameters",
```

is this the number of config parameters in redis? or the config parameters requested?
i think N*M

---

N and M being..?

if any of them is the number of configurations in redis, remember it's constant

---

yes, realized that later, so i suggest
```suggestion
        "complexity": "O(N) when N is the number of configuration parameters provided",
```